conftest.py

```diff
@@ -110,3 +120,38 @@ def dummy_newdle(db_session, dummy_uid):
     db_session.add(newdle)
     db_session.flush()
     return newdle
+
+
+@pytest.fixture
+def multiple_dummy_newdles(db_session, dummy_uids):
```

Comment:
A `create_newdle` fixture  that returns a function creating a dummy newdle would be more flexible.. so you could do something like this in your test:

```python
def test_whatever(create_newdle):
    newdle1 = create_newdle(1)
    newdle2 = create_newdle(2)
```

At least that's what we do [in indico](https://github.com/indico/indico/blob/master/indico/testing/fixtures/event.py).

If also removes the need for a `dummy_uids` fixture (you can default to the `dummy_uid` and just provide a custom uid if you want a different one)

---

In any case, nothing wrong with your option so if you want to keep it like this for now I'm fine with it as well.

Reply:
Added, sounded like a better idea. Thanks.